tap_hubspot/__init__.py

```diff
@@ -319,6 +361,19 @@ def gen_request(STATE, tap_stream_id, url, params, path, more_key, offset_keys,
         while True:
             data = request(url, params).json()
 
+            if v3_fields:
+                url = get_url('deals_v3_search')
+                body = {"properties": v3_fields, "limit": 100}
+                v3_data = post_search_endpoint(url, body)
+                additional_fields = {}
+                for item in v3_data.json()['results']:
+                    # We nest `y` under 'value' in order to match the
+                    # schema and the shape of other fields in 'properties'
+                    additional_fields[int(item['id'])] = {x:{'value': y} for x,y in item['properties'].items()
+                                                          if ('hs_date_entered' in x or 'hs_date_exited' in x)}
```

Comment:
I'm slightly confused. For every key in item['properties'] check if 'hs_date_entered' is "in" the key? Is the key a list of strings? Or is it a long string and 'hs_date_entered' could be somewhere inside that string?

Reply:
> 'hs_date_entered' could be somewhere inside that string

This one